### ISSUE TITLE
[bug] Save file fails with db error

### ISSUE THREAD

**Issue by maximivanov** [directus/directus#1160]
## Version Info
  - Directus version: build (59c5fe39b)
  - PHP version: 5.6.21
  - MySQL version: 5.7.12
  - Web server: nginx 1.8.1
  - OS name and version: OS X El Capitan 10.11.5

## Expected Behavior
Upload and save new file (jpg image)

## Actual Behavior
Server error:

```
class: "RuntimeException"
code: 0
file: "/Users/maxivanov/Projects/directus/api/core/Directus/Db/TableGateway/AclAwareTableGateway.php"
line: 530
message: "This query failed: INSERT INTO `directus_files` (`active`, `name`, `title`, `size`, `type`, `charset`, `width`, `height`, `location`, `caption`, `tags`, `user`, `date_uploaded`, `storage_adapter`) VALUES ('1', 'enhanced-buzz-wide-19115-1413712175-16.jpg', 'Enhanced Buzz Wide 19115 1413712175 16', '150580', 'image/jpeg', 'binary', '990', '673', '', '', '', '1', '2016-05-28 09:50:38 UTC', 'local')"
```

## Steps to Reproduce
 1. After fresh install go to Files -> Upload new file, pick a file and hit (+) to save.

## Logs or Screenshots
I debugged a bit and problem seems to be related to https://github.com/directus/directus/blob/build/api/api.php#L922 which returns date with " UTC" appended. Later on when it tries saving it to mysql it fails with message `Statement could not be executed (22007 - 1292 - Incorrect datetime value: '2016-05-28 09:58:24 UTC' for column 'date_uploaded' at row 1)`

**Comment by WellingGuzman** [comment]
Hi @maximivanov thanks for reporting this. I will on a fix.

For future reference, I have one question, do you have any MySQL modes enabled? on Strict mode I get a different error.  `1264 Out of range value for column 'date_uploaded'` So I wonder which one do you have that gives you that error.

Thanks.

**Comment by maximivanov** [comment]
Sure.

```
SELECT @@GLOBAL.sql_mode;
STRICT_TRANS_TABLES,NO_ENGINE_SUBSTITUTION
```

**Comment by WellingGuzman** [comment]
@maximivanov this https://github.com/directus/directus/commit/2c835d97b194ccc1e0f0a35143508fd940dc6ada should have fixed the problem, and it will be release on v6.2.6
